otx/algorithms/classification/utils/data.py

```diff
@@ -133,11 +133,11 @@ def __init__(
     @staticmethod
     def _load_text_annotation(annot_path, data_dir):
```

Comment:
Could you explain why this should be changed?

Reply:
Ah, I can revert this change. This allows loading multiclass datasets from JSON files, and not only from a directory tree. It was useful to subsample the datasets. I will revert it, sorry.